docs/source/user-guide/tasks/docker-image.rst

```diff
@@ -0,0 +1,137 @@
+
+Quickstart
+----------
+
+#. Build a docker image to run a supplied command
+
+   .. code-block:: text
+
+     anaconda-project dockerize --command <command-name> -t <image name>
+
+#. Run the Docker image and proxy port 8086
+
+   .. code-block:: text
+
+     docker run -p 8086:8086 <image name>
+
+
+Build docker images
+-------------------
+
+By default running the ``dockerize`` command will create a
+Docker image to execute the *default* command.
+
+The *default* command is determined in the following order
+
+#. The command named ``default``
+#. The first command listed in the project file if no command is named ``default``
+
+The ``s2i-anaconda-project`` builder images have Miniconda and ``anaconda-project`` installed. When the ``dockerize``
+command is run the following steps are performed.
+
+#. The project is archived to a temporary directory to ensure that files listed in your ``.projectignore`` (including
+   the local ``envs`` directory) are not copied into the Docker image.
+#. The ``s2i build`` command is run from the temporary directory to construct a new Docker image from the builder image.
+
+The steps in the ``s2i build`` process are
+
+#. The temporary project directory is added to the image.
+#. The `s2i assemble script <https://github.com/Anaconda-Platform/s2i-anaconda-project/blob/master/s2i/bin/assemble>`_ is run to prepare the ``env_spec`` for the desired command.
+#. Conda clean is run to reduce the size of the output Docker image.
+
+
+Options
+^^^^^^^
+
+The ``dockerize`` command accepts several optional arguments
+
+``--command``
+  The named command to execute in the ``RUN`` layer of the Docker image.
+  Default: ``default``
+  See the HTTP commands section below.
+
+``-t`` or ``--tag``
+  The name of the output Docker image in the format ``name:tag``. By default
+  Default: "<project-name>:latest", where <project-name> is taken from the name
+  tag in the anaconda-project.yml file.
+
+``--builder-image``
+  The s2i builder image name to use.
+  Default: ``conda/s2i-anaconda-project-ubi7``
+  By default this is image is pulled from DockerHub when ``dockerize`` is run.
+  See the Custom Builder Image section below to construct your own builder images.
+
+s2i build arguments
+  Any further arguments or those supplied after ``--`` will be given to the ``s2i build`` command.
+  See the `s2i build documentation for available build flags. <https://github.com/openshift/source-to-image/blob/master/docs/cli.md#build-flags>`_
+
+HTTP options
+^^^^^^^^^^^^
+
+When the docker image is run the `s2i run script <https://github.com/Anaconda-Platform/s2i-anaconda-project/blob/master/s2i/bin/run>`_
+is executed with the supplied command. The full run command is
+
+.. code-block:: bash
+
+  anaconda-project run $CMD --anaconda-project-port 8086 --anaconda-project-address 0.0.0.0 --anaconda-project-no-browser --anaconda-project-use-xheaders
+
+This ensures that the command listens over port 8086 if it support the :ref:`http-commands`.
```

Comment:
```suggestion
This ensures that the command listens over port 8086 if it supports the :ref:`http-commands`.
```

Not sure if "listens over" is right; probably should be either "listens to" or "communicates over".